SwrveCommonSDK/src/main/java/com/swrve/sdk/SwrveImp.java

```diff
@@ -897,6 +917,7 @@ protected SwrveConversationCampaign loadConversationCampaignFromJSON(JSONObject
         return new SwrveConversationCampaign((SwrveBase<?, ?>) this, campaignData, assetsQueue);
     }
 
+    @RequiresPermission (Manifest.permission.WRITE_EXTERNAL_STORAGE)
```

Comment:
If we annotate them like this, does it mean it 'always' requires access to the external storage?